brawlcord/data/trophy_road.json

```diff
@@ -560,7 +560,7 @@
     "94": {
         "Trophies": 14500,
         "RewardType": 1,
-        "RewardCount": 150,
+        "RewardCount": 1000,
         "RewardExtraData": ""
```

Comment:
Improved the trophy road. Thinking of adding a compensation for players who already progressed a lot in the TR, might extend the trophy road after this update